src/Language/PureScript/Docs/RenderedCode/RenderType.hs

```diff
@@ -131,15 +131,15 @@ matchType = buildPrettyPrinter operators matchTypeAtom
     OperatorTable [ [ AssocL typeApp $ \f x -> f <> sp <> x ]
                   , [ AssocR appliedFunction $ \arg ret -> mintersperse sp [arg, syntax "->", ret] ]
                   , [ Wrap constrained $ \deps ty -> renderConstraints deps ty ]
-                  , [ Wrap forall_ $ \tyVars ty -> mconcat [keywordForall, sp, mintersperse sp (map typeVar tyVars), syntax ".", sp, ty] ]
+                  , [ Wrap forall_ $ \tyVars ty -> mconcat [keywordForall, sp, mintersperse sp (map (\(v, mbK) -> maybe (typeVar v) (\k -> mintersperse sp [mconcat [syntax "(", typeVar v], syntax "::", mconcat [renderKind k, syntax ")"]] ) mbK) tyVars), syntax ".", sp, ty] ]
```

Comment:
This is a really long line. Maybe we can split up the rendering for the type variable into another function.

Reply:
Good point - I got a little carried away there 😆